lib/spack/spack/cmd/spec.py

```diff
@@ -80,11 +80,7 @@ def spec(parser, args):
         print("--------------------------------")
         print(spec.tree(**kwargs))
 
-        print("Normalized")
-        print("--------------------------------")
-        spec.normalize()
-        print(spec.tree(**kwargs))
-
+        kwargs['hashes'] = args.long or args.very_long
```

Comment:
I think this is good but it would make sense to move the above setting:

`'hashes': False,  # Always false for input spec`

Into this loop as well. That is clearer and I think that without that the hashes will be printed for the next input spec.

Reply:
Totally correct.

Reply:
@scheibelp Done. Should be good to go.